pages/pet-detail-page/[id].js

```diff
@@ -345,6 +345,11 @@ export default function PetDetailPage({
             <MoneyColored cost={200} money={userStats.money} /> <MoneyImage />
           </StyledButton>
         )}
+        <StyledButton
+          onClick={() => router.push(`/${id}/minigames/merge-pets/`)}
+        >
+          Merge Pets
+        </StyledButton>
```

Comment:
The button is only for now, later there will be a kind of pop-up where you can choose between all the mini-games. And to make it look uniform at the moment, I'm using the button here instead of a link.